noteable_magics/datasource_postprocessing.py

```diff
@@ -295,13 +296,24 @@ def postprocess_databricks(
         if connect_file_path.exists():
             connect_file_path.unlink()
 
-        # Now let databricks-connect external command (re)build it and do whatever
-        # else it does. See ENG-5517. The 'y' at the start accepts the license agreement.
-        # (We've fallen oh so far from Don Libes' tcl Expect for stuff like this.)
-        pipeline = f"echo y {args['host']} {args['token']} {args[cluster_id_key]} {args['org_id']} {args['port']} | databricks-connect configure"
-        os.system(pipeline)
+        p = Popen(['databricks-connect', 'configure'], stdout=PIPE, stdin=PIPE, stderr=PIPE)
+        _stdout, stderr = p.communicate(input=f"""y
```

Comment:
Should we add a timeout arg, so that it doesn't hang forever if anything goes awry with the `databricks-connect`?